tests/test_utils/test_diskspeed.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python3 -OO
+# Copyright 2007-2019 The SABnzbd-Team <team@sabnzbd.org>
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+"""
+tests.test_utils.test_diskspeed - Testing sabnzdb diskspeed
+"""
+
+from sabnzbd.utils.diskspeed import diskspeedmeasure
+import sys
+
+class TestDiskSpeed:
+
+    def test_disk_speed(self):
+        if "linux" in sys.platform:
+            dir = "/media/sander/"
+
+        elif "win32" in sys.platform:
+            dir = "D:/"
+
+        elif "darwin" in sys.platform:
+            dir = "/Volumes/CARTUNES/"
+
+        speed = diskspeedmeasure(dir)
+        # TODO : Provide a valid directory
```

Comment:
You can just use 
```python
import os
cwd = os.getcwd()
```
Directory doesn't really matter, since it should just be able to run anywhere.
After the test you should check if the `outputTESTING.txt` file is not present in the directory.

Reply:
Changes done.